pkg/kubectl/cmd/config/get_context.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	"k8s.io/kubernetes/pkg/client/unversioned/clientcmd"
+	clientcmdapi "k8s.io/kubernetes/pkg/client/unversioned/clientcmd/api"
+	"k8s.io/kubernetes/pkg/kubectl"
+
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+type GetContextOptions struct {
+	configAccess clientcmd.ConfigAccess
+	contextName  string
+}
+
+const (
+	get_context_long = `Displays one or many contexts from the kubeconfig file.`
+
+	get_context_example = `
+  $ kubectl config get-context
+    NAME                 CLUSTER         AUTHINFO     NAMESPACE
```

Comment:
@kargakis remind me.  We don't usually demonstrate output in the examples, right?